src/hooks/create_move.cpp

```diff
@@ -54,6 +54,9 @@ namespace hooks
 		if (settings::misc::fast_stop)
 			features::fast_stop(cmd);
 
+		if(settings::misc::music)
+			color_modulation::music_kit_changer();
```

Comment:
Running the code every cmove frame isnt good idea.

Reply:
At least it works. I tried it, it works well